samples/secure-sql-chatbot/src/main/java/io/quarkiverse/langchain4j/sample/chatbot/MovieMuseSupport.java

```diff
@@ -0,0 +1,33 @@
+package io.quarkiverse.langchain4j.sample.chatbot;
+
+import jakarta.inject.Singleton;
+
+import dev.langchain4j.service.SystemMessage;
+import dev.langchain4j.service.UserMessage;
+import io.quarkiverse.langchain4j.RegisterAiService;
+
+@RegisterAiService
+@Singleton
+public interface MovieMuseSupport {
+
+    @SystemMessage("""
+            Create a SQL query to retrieve the data necessary to
+            answer the user's question using data from the database.
+            The database contains information about top rated movies from IMDB.
+            The dialect is PostgreSQL and the relevant table is called 'movie'.
+            Always include `movie_name` in the SELECT clause.
+
+            The user might have not provided the movie name exactly, in that case
+            try to correct it to the official movie name, or match it using a LIKE clause.
+
+            The table has the following columns:
+            {schemaStr}
+            
+            Sort the list of movies by the genre prefered by the movie watcher: 
+            {genre}
```

Comment:
```suggestion
            {preferredGenre}
```

This has to match the argument name, otherwise, this throws an exception that is swallowed and the bot just doesn't respond, without any error indication...

Reply:
@jmartisk isn't ^ something that would need fixing?

Reply:
@gsmet It is fixed in the current state, maybe you're looking at an outdated commit

Reply:
I was more thinking of this behavior: `otherwise, this throws an exception that is swallowed and the bot just doesn't respond, without any error indication...` but I think I misunderstood and it was all related to this PR not to a Quarkus LangChain4j misbehavior.

Reply:
That we fixed by catching the exception in the `ChatbotWebSocket` class